scripts/config/!msu.nut

```diff
@@ -3,5 +3,8 @@
 	ID = "mod_msu",
 	Name = "Modding Standards & Utilities (MSU)",
 	VanillaID = "vanilla",
-	Class = {}
+	Class = {},
+	BBClass = {
+		Dummy = "scripts/config/msu/classes/dummy_bb_class"
```

Comment:
Perhaps just put it in the Class table with DummyBBClass as the name?

Reply:
No, I think a separate BBClass namespace is much better - as both classes are instantiated in different ways e.g. `local myNewSQClass = ::MSU.Class.WeightedContainer()` vs `local myNewBBClass = ::new(::MSU.BBClass.SomeClass)`.